pkg/operatorsource/configuring_test.go

```diff
@@ -66,13 +68,15 @@ func TestReconcile_NotConfigured_NewCatalogConfigSourceObjectCreated(t *testing.
 		TargetNamespace: opsrcIn.Namespace,
 		Packages:        packages,
 	}
-	kubeclient.EXPECT().Create(context.TODO(), cscWant).Return(nil)
 
 	opsrcGot, nextPhaseGot, errGot := reconciler.Reconcile(ctx, opsrcIn)
 
 	assert.NoError(t, errGot)
 	assert.Equal(t, opsrcIn, opsrcGot)
 	assert.Equal(t, nextPhaseWant, nextPhaseGot)
+
+	errClient := fakeclient.Get(ctx, namespacedName, cscWant)
```

Comment:
Here we want to verify that the function being tested passes the expected `CatalogSourceConfig` object to the `fakeClient`. I suggest the following:
```
cscGot := &v1alpha1.CatalogSourceConfig{}
errClient := fakeclient.Get(ctx, namespacedName, cscGot)
assert.NoError(t, errClient)
assert.Equal(t, cscWant, cscGot)
```
This will ensure that the expected value of `CatalogSourceConfig` was passed to `fakeClient.Create`

Reply:
So the only change here is to add an `assert.Equal` and move the `csc` declaration closer to the `fakeclient` call, correct?

Reply:
I've addressed these changes following your suggestion.